src/MastodonConnectionHelper.cs

```diff
@@ -20,7 +20,32 @@ public static async Task<List<string>> GetFollowedTagsAsync()
         await using var reader = await cmd.ExecuteReaderAsync();
         while (await reader.ReadAsync())
             res.Add(reader.GetString(0));
+        reader.Close();
+        cmd.Dispose();
```

Comment:
they're both already being disposed thanks to using, no?

Reply:
I had initially code like:
```
    await using var conn = new NpgsqlConnection(Config.Instance.MastodonPostgresConnectionString);
    await conn.OpenAsync();

    var res = new List<string>();

    // Add the new query to get the values for the `res` list
    await using var cmd = new NpgsqlCommand("SELECT DISTINCT col->'params'->>'id' FROM web_settings, json_array_elements(data->'columns') col WHERE col->>'id' = 'HASHTAG' AND col->'params'->>'id' IS NOT NULL", conn);
    await using var reader = await cmd.ExecuteReaderAsync();
    while (await reader.ReadAsync())
        res.Add(reader.GetString(0));

    // Get the values for the `res` list from the original query
    await using var cmd2 = new NpgsqlCommand("SELECT DISTINCT tags.name FROM tag_follows JOIN tags ON tag_id = tags.id ORDER BY tags.name ASC;", conn);
    await using var reader2 = await cmd2.ExecuteReaderAsync();
    while (await reader2.ReadAsync())
        res.Add(reader2.GetString(0));
 ```
 
 This cause an error: `Npgsql.NpgsqlOperationInProgressException (0x80004005): A command is already in progress`
 
 Thats why I added the close and dispose, but now they are in two voids, I will remove them

Reply:
ah right, you could have closed the using and that should have taken care of it